README.md

```diff
@@ -1,8 +1,58 @@
 # React Native Animation Workshop
-This project contains all the information to implement the different animations of the React Native Workshop in BitsConf 2019.
+Welcome to the workshop! You will be implementing some animations using React Native. Before you do that, we need to get you set up.
 
-# TODO
-* Add setup instructions
-* Include videos or GIFs of the animations to be implemented
-* Include references to the documentation most relevant for this
-* Include link to the slides so the sample code is available
+We **recommend that you use a MAC** to run the workshop or pair up with someone who has a MAC.
+
+Any problems, questions or comments you may have, please Slack us: @juanjo and @alberto.chamorro.
+
+# Setup
+## Repo and Dependencies
+1. You need [yarn](https://yarnpkg.com). If you want to cehck if you have it, run `yarn --version`. If you have version 1.15 or above, you're good. If you don't, install yarn using homebrew running `brew install yarn` in your terminal window.
+If you don't have homebrew you can get it [here](https://brew.sh/).
+2. You need [watchman](https://facebook.github.io/watchman/docs/install.html). If you don't have it, install it with `brew install watchman`. If you want to check whether or not you have it type `watchman -v`. If you have version 4.9.4 or above, you don't need to do anything. If you don't, upgrade it with `brew upgrade watchman`.
+3. You need [expo](https://expo.io/). Expo allows you to build, deploy and iterate on your React Native app very quickly. Install expo globbally using `yarn globall add expo-cli`.
+Run `expo --version` on your terminal window to ensure it installed correctly. You should see something like 1.1.0 (or the version you installed).
+4. You need to clone this repo.
+5. Finally, to install all the dependencies this project requires, go to the folder where you cloned the repo and run `yarn install`.
+
+## IDE
+We recomnend [Visual Studio Code](https://code.visualstudio.com/). It's also a widely used option at Hudl.
+
+## Running the app
+Congratulations, if you've made it here you have your development environment ready :tada:. Now, you need a place to deploy the fancy animations you'll be creating.
+
+There are 2 options: Running it on the device and on the simulator. Ideally, we could all have them running on the devices. That requires your phone and computer to be on the same network. However, since we don't know the quality of network we'll have on the venue, we highly recommend that you also install one of the simulators as a fallback option in case network isn't good enough.
+
+## Option 1: Run it on your iOS or Android simulator.
+1. Install iOS or Android simulators, if you don't have them already:
+    1. For iOS, [install Xcode](https://itunes.apple.com/es/app/xcode/id497799835?l=en&mt=12).
```

Comment:
You need to install the simulators too, not just Xcode. It's a huge download, so it's important that the attendands have them downloaded already. Probably the command line tools too..

Reply:
I _think_ Xcode comes with some simulators by default, doesn't it? I believe we also download older versions to test Modi in all supported versions but for the workshop I think it'll be fine with what it brings by default.

I'll add a note about command line tools though.

Reply:
yeah, that sounds familar